tools/pipelines/test-real-service.yml

```diff
@@ -137,7 +137,22 @@ jobs:
       inputs:
         command: 'custom'
         workingDir: packages/test/end-to-end-tests
-        customCommand: 'run ci:test:realsvc'
+        customCommand: 'run test:realsvc:routerlicious:report'
+
+    - task: Npm@1
+      displayName: '[end-to-end tests] npm run test:realsvc:odsp:report'
+      continueOnError: true
+      timeoutInMinutes: 20
```

Comment:
currently this pipeline times out after 60 min. so giving each 20 min for now. this can always be tuned